routes/road-properties.js

```diff
@@ -270,6 +273,34 @@ function deleteHandler(req, res) {
   });
 }
 
+
+function patchPropertyHandler(req, res) {
+  if (validate(req.payload) !== undefined) {
+    return res(Boom.badData());
+  }
```

Comment:
Another check you may want to add is that the JSON object in the request is of depth 1, and doesn't have any nested objects/arrays.

Reply:
added